src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -5,16 +5,21 @@
 import codeu.model.data.Conversation;
 import codeu.model.data.User;
 import codeu.model.store.basic.UserStore;
+
```

Comment:
I think convention goes that we don't leave lines between different imports? But this is trivial